lib/components/src/ActionBar/ActionBar.stories.tsx

```diff
@@ -28,7 +28,7 @@ export default {
   ],
 };
 
-export const singleItem = () => <ActionBar actionItems={[{ title: 'Clear', onClick: action1 }]} />;
```

Comment:
this space removal doesn't seem intentional. or is it?

Reply:
It wasn't, will be back in https://github.com/storybooks/storybook/pull/6500 😉